packages/server/src/middleware/joi-validator.js

```diff
@@ -2,9 +2,16 @@ function validate(schema, property) {
   // Return a Koa middleware function
   return (ctx, next) => {
     if (schema) {
-      const { error } = schema.validate(ctx[property])
+      let params =
+        ctx[property] != null
+          ? ctx[property]
+          : ctx.request[property] != null
```

Comment:
NAB: nested ternaries don’t read well. Might be better to split the separate conditions out into variables.

Reply:
Yeap you're right, probably for the best!